### ISSUE TITLE
Develop FDA STEF Examples

### ISSUE THREAD

**Issue by drewcdisc** [cdisc-org/analysis-results-standard#189]

**Comment by ASL-rmarshall** [comment]
The only FDA STF example that will be included with ARS v1 (and the hackathon) is "Table 2 Baseline Demographic and Clinical Characteristics, Safety Population, Pooled Analyses (or Trial X)". This table does not contain any comparative statistics (despite the footnote).  The CDISC Pilot data will be used (again) as the basis for the example. 
 Relevant summary statistics will be shown for the two active treatments (Xanomeline Low Dose, Xanomeline High Dose), Placebo and Total Population.  The Active Control column will be excluded because there is not active control in the pilot study.  The following will be applied for each characteristic:
- **Sex**: as shown.
- **Age**: as shown.
- **Age groups**: as shown - age groups will be recalculated based on age and the four groups in the FDA display will be used (including the overlapping groups).
- **Race**: as shown - only the groups found in the data will be shown (there will be no "Native Hawaiian or Other Pacific Islander" or "Other" categories).
- **Ethnicity**: as shown - only the groups found in the data will be shown (there will be no "Unknown" category). Also, "Hispanic or Latino" will be used instead of "Hispanic" to reflect the data accurately.
- **Country of participation**: will not be included because:
  1. There is no country variable in pilot ADSL dataset.
  2. The pilot SDTM DM dataset shows that all subjects were from USA anyway.
- **Clinical baseline characteristics**: will not be included because none were included in the corresponding Common Safety Display example.
 
@bhavinbusa Please confirm that you're happy with this plan.

**Comment by ASL-rmarshall** [comment]
@bhavinbusa The FDA STF example is now available for review in the workfiles/examples/Hackathon folder.  For the age groups analysis, I created and used two separate, non-overlapping age groupings (one for <65 and 65+ and another "age sub-group" for 17-<65, 65-<75 and 75+).  Overlapping groupings within a grouping don't really make sense and actually make it difficult to program the analysis.  I think non-overlapping groups should be a recommendation at least, and possibly even a requirement.  Please take a look at the example and let me know what you think.

**Comment by bhavinbusa** [comment]
I agree with you. Looks good. Thxs for pulling this together prior to the
Hackathon.
